src/data/emacs-mode/agda-input.el

```diff
@@ -179,1114 +264,473 @@ order for the change to take effect."
-  ("d"  . ,(agda-input-to-string-list "↓⇓⤋⟱⇊⇵↧⇩↡⇃⇂⇣⇟         ↵↲↳➥ ↯                                                "))
-  ("ud" . ,(agda-input-to-string-list "↕⇕   ↨⇳                                                                    "))
-  ("lr" . ,(agda-input-to-string-list "↔⇔         ⇼↭⇿⟷⟺↮⇎⇹                                                        "))
-  ("ul" . ,(agda-input-to-string-list "↖⇖                        ⇱↸                                               "))
-  ("ur" . ,(agda-input-to-string-list "↗⇗                                         ➶➹➚                             "))
-  ("dr" . ,(agda-input-to-string-list "↘⇘                        ⇲                ➴➷➘                             "))
-  ("dl" . ,(agda-input-to-string-list "↙⇙                                                                         "))
-
-  ("l-"  . ("←"))  ("<-"  . ("←"))  ("l="  . ("⇐"))  ("<="  . ("⇐"))
-  ("r-"  . ("→"))  ("->"  . ("→"))  ("r="  . ("⇒"))  ("=>"  . ("⇒"))
-  ("u-"  . ("↑"))                   ("u="  . ("⇑"))
-  ("d-"  . ("↓"))                   ("d="  . ("⇓"))
-  ("ud-" . ("↕"))                   ("ud=" . ("⇕"))
-  ("lr-" . ("↔"))  ("<->" . ("↔"))  ("lr=" . ("⇔"))  ("<=>" . ("⇔"))
-  ("ul-" . ("↖"))                   ("ul=" . ("⇖"))
-  ("ur-" . ("↗"))                   ("ur=" . ("⇗"))
-  ("dr-" . ("↘"))                   ("dr=" . ("⇘"))
-  ("dl-" . ("↙"))                   ("dl=" . ("⇙"))
-
-  ("l==" . ("⇚"))  ("l-2" . ("⇇"))                   ("l-r-" . ("⇆"))
-  ("r==" . ("⇛"))  ("r-2" . ("⇉"))  ("r-3" . ("⇶"))  ("r-l-" . ("⇄"))
-  ("u==" . ("⟰"))  ("u-2" . ("⇈"))                   ("u-d-" . ("⇅"))
-  ("d==" . ("⟱"))  ("d-2" . ("⇊"))                   ("d-u-" . ("⇵"))
-
-  ("l--"  . ("⟵"))  ("<--"  . ("⟵"))  ("l~"  . ("↜" "⇜"))
-  ("r--"  . ("⟶"))  ("-->"  . ("⟶"))  ("r~"  . ("↝" "⇝" "⟿"))
-  ("lr--" . ("⟷"))  ("<-->" . ("⟷"))  ("lr~" . ("↭"))
-
-  ("l-n"  . ("↚"))  ("<-n"  . ("↚"))  ("l=n"  . ("⇍"))
-  ("r-n"  . ("↛"))  ("->n"  . ("↛"))  ("r=n"  . ("⇏"))  ("=>n"  . ("⇏"))
-  ("lr-n" . ("↮"))  ("<->n" . ("↮"))  ("lr=n" . ("⇎"))  ("<=>n" . ("⇎"))
-
-  ("l-|"  . ("↤"))  ("ll-" . ("↞"))
-  ("r-|"  . ("↦"))  ("rr-" . ("↠"))
-  ("u-|"  . ("↥"))  ("uu-" . ("↟"))
-  ("d-|"  . ("↧"))  ("dd-" . ("↡"))
-  ("ud-|" . ("↨"))
-
-  ("l->" . ("↢"))
-  ("r->" . ("↣"))
-
-  ("r-o" . ("⊸"))  ("-o"  . ("⊸"))
-
-  ("dz" . ("↯"))
-
-  ;; Ellipsis.
-
-  ("..." . ,(agda-input-to-string-list "⋯⋮⋰⋱"))
-
-  ;; Box-drawing characters.
-
-  ("---" . ,(agda-input-to-string-list "─│┌┐└┘├┤┬┼┴╴╵╶╷╭╮╯╰╱╲╳"))
-  ("--=" . ,(agda-input-to-string-list "═║╔╗╚╝╠╣╦╬╩     ╒╕╘╛╞╡╤╪╧ ╓╖╙╜╟╢╥╫╨"))
-  ("--_" . ,(agda-input-to-string-list "━┃┏┓┗┛┣┫┳╋┻╸╹╺╻
+  ;; The following list has been split into multiple smaller lists
+  ;; to avoid triggering a stack overflow during evaluation, on
+  ;; systems with a lower `max-lisp-eval-depth'.
+  (append
+;;;;; Equality and similar symbols.
+   `(("eq" ,(agda-input-to-string-list "=∼∽≈≋∻∾∿≀≃⋍≂≅ ≌≊≡≣≐≑≒≓≔≕≖≗≘≙≚≛≜≝≞≟≍≎≏≬⋕＝"))
```

Comment:
Has the `.` after`"eq"` been removed by purpose?  
Command `C-u C-x =` seems broken, I am getting error:
```
quail-gen-decode-map1: Wrong type argument: fixnump, ("=" "∼" "∽" "≈" "≋" "∻" "∾" "∿" "≀" "≃" ...)
```
This lead me here.